dev/src/tests/fields/lexical-editor-with-multiple-blocks.test.ts

```diff
@@ -668,6 +668,7 @@ describe('Lexical editor with multiple blocks', () => {
       collection: 'policies',
       data: {
         title: 'Test policy',
+        content: fixture2,
```

Comment:
Note that `PAYLOAD_CROWDIN_SYNC_ALWAYS_UPDATE=true` is not active in tests, so they have been updated to ensure fields are changed and an update attempt is made by the plugin.